src/auth/schemas.py

```diff
@@ -0,0 +1,35 @@
+import uuid
+from pydantic import BaseModel, EmailStr
+from datetime import datetime
+from typing import Optional
+
+
+class NewUserCred(BaseModel):
+    email: EmailStr
+    password: str
+
+    class Config:           #pydantic работает только с dict, эта строчка позволяет работать с любым типом
```

Comment:
remove comments like that, at least put them not in russian
orm_mode is needed for other goals, in your case that schema only validate http request, not need to switch orm_mode in True